flair/datasets/biomedical.py

```diff
@@ -5089,7 +5089,7 @@ def entity_type_predicate(member):
 
                 self.huner_corpora.append(corpus)
             except Exception:
```

Comment:
Catching bare Exceptions is considered bad practice, however this code is not changed by this PR.

Reply:
Refactored to catch each exception type separately.

Reply:
I appreciate the effort, thanks a lot!